### PR TITLE
Added the criminal management console

### PR DESCRIPTION
**Pull request by Cruix**
The problem with security is that nobody plays the small but important role of <strike>xenobiology</strike> looking through the security camera consoles, because it is tedious and boring. In some rounds, based on the RNG of <strike>slimes mutating</strike> antagonists showing up, it can take up to two hours to <strike>get rainbow slimes and turn into an immortal changeling</strike> fill the permabrig up with criminals, due to the time it takes to <strike>toss monkeys in the disposal and drag slimes 1-3 tiles out of the pens</strike> locate and apprehend potential criminals, only a few of which will be permabrigable antags.

The criminal management console is a simple quality of life feature that will make this tedious job bearable, increasing the impact that security can have on <strike>themselves because they almost never share all the cool shit they get</strike> the now criminal-free station. Just based on the fact that security officers will no longer have to deal with <strike> slimes moving around making it hard to drag the one you want out</strike> shitlers pulling their prisoners away in the halls, this feature should be speedmerged for the sanity of everyone who plays security.

You might be thinking, "but what about the slight chance that a security officer <strike>gets eaten by a slime</strike> gets outrobusted by a criminal, shouldn't we leave that possibility in to maintain the 'screaming metal deathtrap' theme of space station 13?" Well, honestly, how often does that really happen? Security officers have <strike>biosuits and extinguishers that make them virtually immune to slimes</strike> tasers and flashbangs to easily subdue almost any target at a distance, so the possibility of them screwing up and dying is so remote that we might as well ignore it. 

I look forward to the days when everyone who plays sec will call dibs on security camera duty, and <strike>murder their coworkers to get at that sweet console</strike> respectfully share the duty and not create tons of tickets about people stealing their sub-department job <strike>every second highpop round I mean come on</strike>.

:cl:
rscadd: Security camera consoles have been updated, and can now stun, cuff, and transport criminals to the brig.
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
the hell

**Comment by GrayRachnid** [comment]
👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌

**Comment by ShadowDeath6** [comment]
slimes don't talk back and complain with you hit them though, so there's a lot more generic unfun shit like le trying to pull slimes through one another and failing to do so

File: code/modules/research/xenobiology/xenobio_camera.dm [hunk 1]
```diff
@@ -179,3 +179,190 @@
...
+		C.client.images -= remote_eye.user_image
+		for(var/datum/camerachunk/chunk in remote_eye.visibleCameraChunks)
+			C.client.images -= chunk.obscured
+	C.remote_control = null
+	C.unset_machine()
+	src.Remove(C)
+
+/datum/action/innate/taze
+	name = "Stun"
+	button_icon_state = "monkey_up"
```
Comment (ShadowDeath6) [inline code/modules/research/xenobiology/xenobio_camera.dm:266]: is this a reference

**Comment by drderp3635** [comment]
...uh guys the meme zone tag isn't on this you are scaring me here

**Comment by Time-Green** [comment]
Why would you add the memezone tag?

**Comment by ghost** [comment]
Hilarious

But in all seriousness a way to gulag prisoners would be nice.

**Comment by Nickvr628** [comment]
Can you just leave prisoners in the camera transport limbo? Sounds amazing. INB4 clown gets deposited into singlo for punishment.

Also you can just pick up the cultists and drop them in space, or the burn chamber, or the atmos storage tanks with a stupidly high pressure.

**Comment by Gintoki1** [comment]
I would like to see this

**Comment by Cruix** [comment]
You can't pick them up and hold them, it instantly teleports them to waypoints preplaced on the map. If you really wanted to you could fill up the cells with burning plasma to kill your prisoners instantly.

**Comment by doerakpoes** [comment]
The moment this gets merged I am gonna test it

**Comment by Wilchenx** [comment]
👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌there👌👌👌 right✔there ✔✔if i do ƽaү so my self 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠOOOOOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shit

**Comment by Cruix** [comment]
qwszjn4

**Comment by Cruix** [comment]
qwszjn4

**Comment by ShadowDeath6** [comment]
adds a meme feature mocking people who want slime management console

people unironically want it merged

**Comment by Oakboscage** [comment]
This is a fucking joke right? Someone tell me this isn't real.

**Comment by Stealthkibbler** [comment]
This is the real way to prove a point to players and coders, the goofball way.

**Comment by doerakpoes** [comment]
If this is a joke then why I am too unrobust for normal security man

**Comment by X-TheDark** [comment]
Actually, I'm still thinking whether or not I want this merged.

**Comment by X-TheDark** [comment]
Well played.

**Pull request merged**